sound/soc/sof/ipc4-topology.c

```diff
@@ -1976,12 +1978,36 @@ static int sof_ipc4_control_setup(struct snd_sof_dev *sdev, struct snd_sof_contr
 	return 0;
 }
 
+static int sof_ipc4_pipeline_send_kcps(struct snd_sof_dev *sdev, struct snd_sof_widget *swidget,
+				       __s32 adjust)
```

Comment:
Why `__s32` and not just `s32`?

Reply:
because it's an external parameter, a part of an ABI